plugins/lookup/bitwarden.py

```diff
@@ -109,10 +114,19 @@ def get_field(self, field, search_value, search_field="name"):
         """
         matches = self._get_matches(search_value, search_field)
 
-        if field:
+        if field in ['password', 'passwordRevisionDate', 'totp', 'uris', 'username']:
```

Comment:
Will login never contain anything else than these five fields? (If it does, this PR breaks backwards compatibility.)

Reply:
You're very welcome. Changelog fragment has been added.

I can't answer that definitively, as I can't find the object definition in the source. However, there are no fields in the client that are not represented in my example record above, so I would imagine there are no other "login" fields at this time.

Reply:
Hmm, the problem is that if there are more fields, this would be a breaking change since if anyone was querying these fields successfully in the past, it won't work for them anymore.

Reply:
I dug through the source a bit. I still couldn't find where it's defined on the server side, but I did find [this on the official client](https://github.com/bitwarden/clients/blob/master/libs/common/spec/models/domain/login.spec.ts), which seems to suggest there is one more field, `autofillOnPageLoad`. I have updated the PR with this additional field.

Reply:
This is probably a better place to check in the client: https://github.com/bitwarden/clients/blob/master/libs/common/src/models/data/login.data.ts#L6-L11
And on the server side: https://github.com/bitwarden/server/blob/master/src/Api/Models/CipherLoginModel.cs and https://github.com/bitwarden/server/blob/master/src/Api/Models/Response/CipherResponseModel.cs

Reply:
Thank you for that. I'm not a developer, just a sysadmin, so it's difficult for me to understand and navigate larger code bases. Looks like we definitively got all the login fields now.